packages/docs/verite/patterns/travel-rule.md

```diff
@@ -2,53 +2,104 @@
-5. Bob's financial institution constructs message "M2" which is a VC proving control over the beneficiary address.
-6. Bob's FI transmits M2 privately to Alice's FI using the callback URL specified in M1.
-7. Alice's FI receives and verifies the VC in M2, leaving both Alice's FI and Bob's FI (a) certain that each controls the designated addresses and (b) with a private secure network connection established through the specified callback URL.
+   - Request for proof of control of the beneficiary address (e.g. formatted as a Presentation Request or [CACAO receipt][CACAO] of a [SIWX message][])
+   - One or more identifiers for querying and verifying the identity of the custodian initiating the message
+   - A secure callback URL endpoint for replies (e.g. a Presentation Exchange endpoint for receiving Verifiable Presentations)
+4. Additionally, an FI message should contain, if in "**Fast mode**" :
+   - A Proof-of-control of the originating address (formatted as a Verifiable Credential, for example, or even as a ["sign-in with X" off-chain transaction message][SIWX message])
+   - or, if in "**Suspicious mode**":
+   - A secure callback URL endpoint for requesting proof-of-control credentials by M1 message identifier after establishing trust (see below)
+5. Alice's FI publishes M1 to the decentralized bulletin board.
+6. Bob's financial institution observes M1 being emitted in the bulletin board, verifies that the recipient address is in its custody, and responds.
+
+### Part 2: Trust Establishment: i.e. Exchange of Public Credentials
+
+The trust establishment step (which some might think of as an "anti-spam"/"anti-DoS" step to avoid spam or data-collecting crawlers) is not included in the simplified prototype in our repo. There are various prototyping efforts happening in the DIF and elsewhere that may prove adequate for this step, so we are deferring research on it in case open-source specifications get hardened in the meantime for this kind of authentication and public trust registry building.
+
+1. If in "**Suspicious Mode**", Bob's financial institution would attempt to authenticate Alice's FI, the author of M1 before responding. If in "**Fast Mode**, it could **skip to step 3**.
+2. Bob's financial institution would authenticate Alice's FI by the identifier(s) provided in M1. The simplest way of achieving this would be a `did:web` identifier expressing a DID document corresponding to the private key used to sign M1, particularly if said `did:web` identifier could be used to query public registries of regulated financial institutions maintained by trusted intermediaries, competent authorities, or your friendly local trade association and open source standards organization (i.e. Verite!).  Crucially, however, these registries should be multiple and open, to maximize spread and bootstrap an open ecosystem that is inclusive of multiple closed ones and proprietary authorities.
```

Comment:
Should we also add here that both FI's can query for VCs about the other one to gain trust? VC that proof incorporation in a jurisdicion or other ones.

Reply:
added as hypothesis/possibility down the road